docs/it-ams-monitoring-data.md

```diff
@@ -0,0 +1,53 @@
+# IT-AMS Monitoring Data Integration Guide
+
+## Overview
+The TTA Hub integrates with IT-AMS to retrieve and process monitoring data related to Head Start recipients’ reviews and findings. The main steps are:
+- Monitoring review data is securely transferred from IT-AMS.
+- The system automatically imports the data and stores it in the database (PostgreSQL).
+- The findings generate automatic goals for grant recipients to address compliance issues.
+- The HSES also receives this data for program management visibility. This happens in parallel to the TTA Hub's import.
+
+## How the Integration Works
+1. IT-AMS generates XML data files containing monitoring reviews daily.
+2. The TTA Hub downloads the data via SFTP from `cloud.gov`.
```

Comment:
I thought this came from the `ITAMS_MD_HOST` env variable in cloud.gov, which points to an sftp server not hosted by cloud.gov, unless I am mistaken

Reply:
I realize it was unclear. Fixed.